app/src/main/res/layout/fragment_todo.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
```

Comment:
relative layouts don't have orientations, only linear layouts do. guessing this was left over after you changed it :scissors: sors: sors: sors: